src/site/pages/digital/src/utils/BrowserCompatibility.ts

```diff
@@ -0,0 +1,85 @@
+import {Browser} from "core/utils/Browser";
+import { type } from "os";
+
+type compatible = {
+    react_v: boolean;
+    canvas_v:boolean;
```

Comment:
What do these mean? Please follow the naming conventions of the rest of the codebase which is generally camelCase